### PR TITLE
Refactor imports/exports

### PR DESCRIPTION
**Pull request by ctoffanin**
## Describe your changes
- Changed all component's exports to named export (apart for the content templates)
- Change data of maintainer card story
- Added picture and description to maintainers on the about page

## Link to issue

Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [ ] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

<a href="https://gitpod.io/#https://github.com/AccessibleForAll/AccessibleWebDev/pull/199"></a>

### CODE REVIEW

File: components/Layout/Layout.tsx [hunk 1]
```diff
@@ -30,7 +30,10 @@ const Layout = (props: LayoutProps) => {
 				showNavMobile={showNavMobile}
 			/>
 			<div className={styles.layoutContainer}>
-				<NavPrimary activeNavLink={activeNavLink} />
+				<NavPrimary
+					activeNavLink={activeNavLink}
+					handleNavClick={handleNavClick}
```
Comment (EmmaDawsonDev) [inline components/Layout/Layout.tsx:35]: Hmm, handleNavClick doesn't need passing down to NavPrimary as it's only necessary for the mobile view.

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Everything looks good apart from handleNavClick shouldn't be needed in NavPrimary. Thanks for going through everything!

File: components/Layout/Layout.tsx (hunk 1 above)
Comment (ctoffanin) [reply components/Layout/Layout.tsx:35]: Ops, my bad. I meant to remove it from the `NavPrimary` component. It was introduced in a previous PR.

**Review by EmmaDawsonDev — APPROVED** [review]
Brilliant! Thank you :D

**Pull request merged**